src/Accounts/Accounts/help/Connect-AzAccount.md

```diff
@@ -134,6 +134,7 @@ command connects the specified Azure tenant using the service principal credenti
 authenticates as a service principal.
 
 ```powershell
+$SecurePassword = $password | ConvertTo-SecureString -AsPlainText -Force
```

Comment:
Better use the following so that no new variable will be introduced.
```suggestion
$SecurePassword = ConvertTo-SecureString -String "Password123!" -AsPlainText -Force
```